app/jobs/socure_docv_results_job.rb

```diff
@@ -3,41 +3,76 @@
 class SocureDocvResultsJob < ApplicationJob
   queue_as :default
 
-  attr_reader :document_capture_session_uuid
+  attr_reader :document_capture_session_uuid, :async
 
   # @param [String] document_capture_session_uuid
-  def perform(document_capture_session_uuid:)
+  def perform(document_capture_session_uuid:, async: true)
     @document_capture_session_uuid = document_capture_session_uuid
+    @async = async
```

Comment:
Could we set `@async = IdentityConfig.store.ruby_workers_idv_enabled` and avoid passing it in?

Reply:
@amirbey thoughts on this, I am not opposed to it

Reply:
well, i was implementing LG-15015 which is calling perform_now, regardless of the `ruby_workers_idv_enabled` config, which this change no longer allows me accurately capture async. This change inhibits us from using perform_now (ie: [LG-14998](https://cm-jira.usa.gov/browse/LG-14009)) 😐

Reply:
will revert and keep the optional attribute